praw/models/reddit/modmail.py

```diff
@@ -0,0 +1,52 @@
+from ...const import API_PATH, urlparse
+from ...exceptions import ClientException
+from .base import RedditBase
+
+
+class Modmail(object):
```

Comment:
This class should probably live along side the other helper utilities that are associated with `Subreddit`.